### ISSUE TITLE
tapOn taps on the view twice

### ISSUE THREAD

**Issue by sregg** [mobile-dev-inc/maestro#469]
We have a tag component that you can toggle on and off.
when using `tapOn`, maestro seems to tapping on that view twice, one selecting the tag and again unselecting it.
Is there a way to fix that?

Here's the screen record video: https://photos.app.goo.gl/R9JEjvDMnt3fFFpa9

**Comment by Leland-Takamine** [comment]
Yes you can disable this behavior using `retryTapIfNoChange: false`:

```yaml
- tapOn:
    text: your-text
    retryTapIfNoChange: false
```

**Comment by sregg** [comment]
Thanks Leland. I actually fixed the issue by setting a different a11y label for each selected/unselected state (i.e. `Add Layout tag` and `Remove Layout tag`).
But it's great to hear there's also a `retryTapIfNoChange` param.
I'm seriously blown away with this tool.
We spent weeks setting up Appium/Webdriver and it took me 1 hour last night to write 4x less code with Maestro.
At this point, the only blocker for us to migrate is Browserstack support to run on physical devices in the cloud.
